usr/src/uts/common/fs/zfs/zfs_vnops.c

```diff
@@ -653,43 +654,28 @@ zfs_read(vnode_t *vp, uio_t *uio, int ioflag, cred_t *cr, caller_context_t *ct)
 static int
 zfs_write(vnode_t *vp, uio_t *uio, int ioflag, cred_t *cr, caller_context_t *ct)
 {
-	znode_t		*zp = VTOZ(vp);
```

Comment:
Why are you changing the style of the variable declarations here?

Reply:
Since I was adding the kstat_update calls in that functions, I figured I'd refactor the styling here so variables are declared closer to their usage now that C99 is enabled.